pkg/k8shandler/forwarding_test.go

```diff
@@ -10,9 +12,11 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
 	cl "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
-	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
+	loggingv1 "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	loggingv1Alpha "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
```

Comment:
small caps and complete version: `loggingv1alpha1`

Reply:
Fixed